client/webserver/site/src/js/markets.js

```diff
@@ -1243,11 +1246,20 @@ export default class MarketsPage extends BasePage {
     }
   }
 
+  setBalanceVisibility () {
+    if (this.market.dex.connected) {
+      Doc.show(this.page.marketBalance)
+    } else {
+      Doc.hide(this.page.marketBalance)
+    }
+  }
```

Comment:
This seems alright to me but in other places we show/hide the whole order form that includes the balance table (`Doc.{show,hide}(page.orderForm)`), such as when the account is not yet paid.  That actually happens in `setRegistrationStatusVisibility`, which is called right above the `setRegistrationStatusVisibility` call you added. @fernandoabolafio or buck could advise of that's more appropriate or not.

Reply:
I am also hiding the form now. 

I will wait to see if it is ok to check if to handleBalanceNote or setting connection status.

I am fine either way